doc/api/util.md

```diff
@@ -325,6 +325,60 @@ The `--throw-deprecation` command-line flag and `process.throwDeprecation`
 property take precedence over `--trace-deprecation` and
 `process.traceDeprecation`.
 
+## `util.diff(actual, expected)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `actual` {any}
+* `expected` {any}
+* Returns: {string | null} A string if there is difference
+  between `actual` and `expected`, `null` otherwise
+
+[`util.diff()`][] is a utility function that compares two values and returns a
+string that represents the difference between them. It is useful for debugging
+and testing.
+The output is exactly the same as the one used by all the assertion errors, therefore
```

Comment:
Could we add the complexity for space and time as well?
Especially with Myers there are different algorithms and it would help to know which one is used.